.github/workflows/check-change.yml

```diff
@@ -1,8 +1,36 @@
-name: Check change
+name: Check changes/package versions
 on:
   pull_request:
 
 jobs:
+  check-package-versions:
+    runs-on: ubuntu-latest
+    env:
+      SYNCPACK_VERSION: latest
```

Comment:
Is this needed? We fail if the dependency isn't declared on the `package.json`  so then `latest` is never used.

Reply:
Maybe using output would be more clear for the flow as the run step will have a dependency on the previous step and won't run without it.

Reply:
> Is this needed? We fail if the dependency isn't declared on the

good point. I wanted to have his there for better visibility that there is something going on with particular env var

> Maybe using output

what kind of output are you referring to ?